### PR TITLE
Refresh /server/hyperscale

### PR DESCRIPTION
**Pull request by petesfrench**
*this will act as the feature branch for the server updates*

## Done

- Refresh /server/hyperscale based on [copydoc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit) and [design](https://www.figma.com/file/Kw2SwMkH49tN54NbnKs5dt/U.com---server-(rebranding)?type=design&node-id=1-10&mode=design&t=ozrDP1wwHOGlBNCq-0)
- I used [this](https://docs.google.com/document/d/1kq0SPv-PAIfL9nHGxgNDpHhr5OplE2iNaZZiqG4RT48/edit#heading=h.qr5jy1sbfmj) as reference for the contextual footer

## QA

- Check [the demo](https://ubuntu-com-13521.demos.haus/server/hyperscale) matches the [copydoc](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit) and design [design](https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit)
- Test on desktop and mobile views

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8256

### CODE REVIEW

**Comment by juanruitina** [comment]
Looks good, very minor tweaks.

- [ ] Update the [Get more details on Juju here ›](https://jujucharms.com/) label and destination: [Learn more about Juju ›](https://juju.is/) (I think it should point to juju.is instead of charmhub.io)
- [ ] Change the alt text of the logos to the full brand name: Hewlett Packard Enterprise, ARM, Cavium, Applied Micro. Please also put them inside H3 tags: it's fine that they are images, they still act as headings. Please change the alt text of the HP logo in the quote to just "HP". (We should not use words like "logo", "image" or "photo" in alt text, it's redundant)
- [ ] Fix the capitalisation in the "[Get Ubuntu Server](https://ubuntu-com-13521.demos.haus/download/server)" CTA in the footer.
- [ ] 🚨🚨🚨 UBUNTU ADVANTAGE ALERT 🚨🚨🚨  Please change the copy in the footer so we say "Ubuntu Pro" instead of "Ubuntu Advantage"

File: templates/server/hyperscale.html [hunk 1]
```diff
@@ -1,232 +1,387 @@
...
 {% block meta_description %}Unlock the promise of new extreme low-energy server technology{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1f56CCzH519lIVa4KvxDt0QpxYFMK-bMdVlKo4TvPSIk/edit{% endblock meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock body_class %}
+
 {% block content %}
-<div class="p-strip--suru-bottomed">
-  <div class="row u-equal-height">
-    <div class="col-8 u-vertically-center">
+<div class="p-strip">
```
Comment (lyubomir-popov) [inline templates/server/hyperscale.html:10]:
```suggestion
<div class="p-strip is-shallow u-no-padding-bottom">
```

File: templates/server/hyperscale.html [hunk 2]
```diff
@@ -1,232 +1,387 @@
...
       <h1>Ubuntu leads in hyperscale</h1>
-      <p>Ubuntu is the hyperscale OS, natively powering scale-out workloads on a new wave of low-cost, ultra-dense hardware based on x86 and Arm processors.</p>
-      <p><a class="p-button--positive" href="/download/server">Download Ubuntu Server</a></p>
-      <p><a href="/download/arm">Download Ubuntu for ARM&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col">
+      <p>Ubuntu is the hyperscale OS, natively powering scale-out workloads on a new wave of low-cost, ultra-dense hardware based on x86 and ARM processors.</p>
+      <p>
+        <a href="/download/server" class="p-button--positive">Download Ubuntu Server</a>
+        <a href="/download/arm">Download Ubuntu for ARM&nbsp;&rsaquo;</a>
```
Comment (lyubomir-popov) [inline templates/server/hyperscale.html:19]:
```suggestion
        <a href="/download/arm">Download&nbsp;Ubuntu&nbsp;for&nbsp;ARM&nbsp;&rsaquo;</a>
```

File: templates/server/hyperscale.html [hunk 3]
```diff
@@ -1,232 +1,387 @@
...
+          height="70",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+    <div class="col-9 col-medium-5">
+      <div class="p-section--shallow">
+        <p class="p-heading--2"><em>
```
Comment (lyubomir-popov) [inline templates/server/hyperscale.html:93]:
```suggestion
        <p class="p-heading--4"><em>
```

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]
Mostly looks great, a few suggestioins in the code for things that are not quite there yet. HYappy to approve once those are fixed.

**Comment by petesfrench** [comment]
@lyubomir-popov Can you take another look please? Check out how I have handled things on different screen sizes as I made some 'creative' assumptions

**Comment by lyubomir-popov** [comment]
Added some small fixes and an illustration as it seems a bit dry at the top - 

image

@juanruitina is that ok with you?

**Comment by juanruitina** [comment]
@lyubomir-popov Sure thing. The margin at the bottom of the image does look a bit excessive, removing `p-strip` from its container seems to improve it? (It's already within another `p-strip`)

File: templates/server/partial/_server-footer.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<div class="p-section">
```
Comment (mtruj013) [inline templates/server/partial/_server-footer.html:1]: Design for this looks slightly different (heading / "download" in the same row), could you take a look?

**Review by mtruj013 — APPROVED** [review]
Thanks @petesfrench! LGTM with some minor changes

File: templates/server/partial/_server-footer.html (hunk 4 above)
Comment (petesfrench) [reply templates/server/partial/_server-footer.html:1]: Thanks, fixed!

**Comment by petesfrench** [comment]
@lyubomir-popov illustration added, how does it look to you?

**Review by lizzochek — APPROVED** [review]

**Review by lyubomir-popov — APPROVED** [review]

**Pull request merged**